django/test/utils.py

```diff
@@ -640,3 +640,15 @@ def setUp(self):
 
     def tearDown(self):
         self.logger.handlers[0].stream = self.old_stream
+
+
+def add_tags(*tags):
+    """
+    Decorator to add tags to test
+    """
+    def decorator(function):
+        for tag in tags:
+            setattr(function, tag, True)
```

Comment:
I would favor keeping all tags in `tags = set()` attribute on the object to prevent collisions instead.